### PR TITLE
Pretty Events

### LINKED ISSUE #198: Event cards are pretty ugly
Clean up what info we need to show, and add images

screen shot 2015-04-22 at 3 06 36 pm

### LINKED ISSUE #207: Can't remove images from events
If you do, `event_data_from_form` in `app/lib/events` will set `event_image` to `None`, and the field will be stripped by `_remove_none_fields`.

### PR DESCRIPTION
**Pull request by danrschlosser**
- Fixes #207 
- Closes #198 
- Makes events more attractive 
- Cleans up `Event.human_readable_datetime`

screen shot 2015-04-27 at 11 55 40 pm

### CODE REVIEW

File: app/static/js/admin/events/create.js [hunk 1]
```diff
@@ -199,7 +199,7 @@ $(function() {
         $('a[href="#close-modal"]').click();
 
         // Set the hidden input to have the image as it's value
-        $('#event_image').val(filename);
+        $('input[name=event_image]').attr('value', filename);
```
Comment (danrschlosser) [inline app/static/js/admin/events/create.js]: Explanation: http://stackoverflow.com/questions/2109322/cant-set-hidden-input-fields-value-using-jquery

File: app/models/Event.py [hunk 2]
```diff
@@ -265,22 +265,62 @@ def human_readable_time(self):
...
-            format = "%I:%M-"
-        else:
-            format = "%I:%M%p-"
-        output += self.start_time.strftime(format).lstrip("0").lower()
-        output += self.end_time.strftime("%I:%M%p").lower().lstrip("0")
-        return output
+        return '{}-{}'.format(self._human_readable_start_time(),
+                              self._human_readable_end_time())
+
+    def _human_readable_start_time(self):
```
Comment (natebrennand) [inline app/models/Event.py]: I think it would be great if these events had some tests. They seem kinda fragile and I would be afraid to touch them.

Comment (danrschlosser) [inline app/models/Event.py]: Do you mean just the date/time formatting methods?

Comment (natebrennand) [inline app/models/Event.py]: ya, `human_readable_time`, `_human_readable_start_time`, `_human_readable_end_time` & `human_readable_datetime`.

It's probably fine to just do `human_readable_datetime`.

Comment (danrschlosser) [inline app/models/Event.py]: I'll probably do just the non-private methods: `human_readable_datetime`, `human_readable_time`, and `human_readable_date`.

**Comment by danrschlosser** [comment]
@natebrennand added tests.

**Comment by natebrennand** [comment]
That's super robust testing :100: 

(this could probably be ripped out to be a PyPi module and you could publish it!)

:shipit:

**Pull request merged**